### ISSUE TITLE
Clarification on Fine-Tuning Details

### ISSUE THREAD

**Issue by joshchristo** [GAIR-NLP/thinking-with-generated-images#4]
Thank you for the excellent work. I have a couple of quick questions and would really appreciate your clarification.

In the Anole paper, you guys mentioned that "we froze most of Chameleon’s parameters and fine-tuned only the logits corresponding to image token ids in transformer’s output head layer." Does this same approach also apply to Thinking with Generated Images?

Additionally, and apologies if this is a basic question, which part of the implementation determines which layers are frozen or unfrozen? 

Thank you again for your time and for sharing such impactful research.

**Comment by EthanC111** [comment]
Thanks for the question! For Thinking with Generated Images, we conducted full finetuning to ensure better model alignment. The training code is opensourced and feel free to try it!
